tests/CustomDatabaseConnectionActivityModelTest.php

```diff
@@ -26,6 +26,16 @@ public function it_uses_a_custom_database_connection()
         $this->assertEquals($model->getConnectionName(), 'custom_sqlite');
     }
 
+    /** @test */
+    public function it_uses_the_default_database_connection_when_the_one_from_configuration_is_null()
+    {
+        config()->set('activitylog.database_connection', null);
+
+        $model = new Activity();
+
+        $this->assertInstanceOf('Illuminate\Database\SQLiteConnection', $model->getConnection());
```

Comment:
I suggest comparing the connection name or the database name, rather than just the type.

Reply:
@stancl`$model->getConnectionName()` is `null` when not set externally. I think that's the desired behaviour, unless I'm missing something, and that's why I'm looking into the resulting connection.
```php
    // Illuminate\Database\ConnectionResolver
    public function connection($name = null)
    {
        if (is_null($name)) {
            $name = $this->getDefaultConnection();
        }

        return $this->connections[$name];
    }
```